critiquebrainz/db/exceptions.py

```diff
@@ -11,3 +11,7 @@ class NoDataFoundException(DatabaseException):
 class BadDataException(DatabaseException):
     """Should be used when incorrect data is being submitted."""
     pass
+
+class IntegrityError(DatabaseException):
```

Comment:
I'm not sure if that's actually necessary. I'd say that `BadDataException` covers this case. It's also a different level of abstraction. Integrities and constraints are involved when working directly with the database and that shouldn't concern users of the `db` package which would receive this exception.